src/components/look-controls.js

```diff
@@ -77,9 +78,11 @@ module.exports.Component = registerComponent('look-controls', {
     this.onMouseDown = bind(this.onMouseDown, this);
     this.onMouseMove = bind(this.onMouseMove, this);
     this.onMouseUp = bind(this.onMouseUp, this);
-    this.onTouchStart = bind(this.onTouchStart, this);
-    this.onTouchMove = bind(this.onTouchMove, this);
-    this.onTouchEnd = bind(this.onTouchEnd, this);
+    if (this.data.touchEnabled) {
+      this.onTouchStart = bind(this.onTouchStart, this);
+      this.onTouchMove = bind(this.onTouchMove, this);
+      this.onTouchEnd = bind(this.onTouchEnd, this);
+    }
```

Comment:
This event binding should happen regardless of `data.touchEnabled`, because touch events might be enabled or disabled at a later time.

Reply:
Yes, This condition is not necessary. We should bind the methods regardless of the flag

Reply:
Addressed in commit 1d6930b61ac66d080bfe57eada841a032fd6e6e8